### PR TITLE
refactor: Upgrade rc-time-picker for new rc-trigger

### PR DESCRIPTION
**Pull request by afc163**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

ant-design/ant-design#19169

### 💡 Background and solution

升级后没发现任何动画问题。👍

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |       none    |
| 🇨🇳 Chinese |      none     |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/time-picker/style/index.less [hunk 1]
```diff
@@ -183,7 +183,7 @@
   }
 
   &-icon,
-  &-clear {
+  &-clear.@{iconfont-css-prefix} {
```
Comment (afc163) [inline components/time-picker/style/index.less]: 随便把图标颜色修了。

**Comment by shaodahong** [comment]
CI Fail, Please Rebase

**Comment by shaodahong** [comment]
have a problem, scroll some distance and trigger https://deploy-preview-19698--ant-design.netlify.com/components/time-picker-cn/#components-time-picker-demo-size, body scroll to the top

**Comment by afc163** [comment]
Hmmm, it can only be reproduced in production mode.

**Comment by shaodahong** [comment]
it's happened at my local start, and In debugger mode, click trigger propagate this line

https://github.com/ant-design/ant-design/blob/8b6ef59917f50d2465c03b3ac86483632eea14a0/components/modal/Modal.tsx#L18

**Comment by afc163** [comment]
Thx!, I reproduce in Chrome too.

**Comment by afc163** [comment]
Fixed in https://github.com/react-component/time-picker/commit/6acda64ef3a8bb773acc2108e61c5e02c1c7ce05

**Comment by shaodahong** [comment]
still have

https://deploy-preview-19698--ant-design.netlify.com/components/time-picker-cn/#components-time-picker-demo-basic

1. scroll distance
2. open time panel and select `01:01:01`
3. close and reopen
3. body scroll to the top

if you not reproduce, you can try more times

**Comment by afc163** [comment]
I can't reproduce it now.

**Comment by shaodahong** [comment]
Kapture 2019-11-13 at 13 54 12

**Review by shaodahong — APPROVED** [review]
LGTM

File: components/time-picker/index.tsx [hunk 2]
```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
 import * as moment from 'moment';
 import omit from 'omit.js';
-import { polyfill } from 'react-lifecycles-compat';
-import RcTimePicker from 'rc-time-picker/lib/TimePicker';
+import RcTimePicker from 'rc-time-picker/es/TimePicker';
```
Comment (zombieJ) [inline components/time-picker/index.tsx]: antd-tools 里会换掉，这里不用改

Comment (afc163) [reply components/time-picker/index.tsx]: ok

**Pull request merged**